### DISCUSSION TITLE
Styling invalidated cells

### DISCUSSION THREAD

**Discussion by yurivish** [observablehq/framework#1605] [Q&A] [answered]
In Observable [notebooks](https://observablehq.com/d/ff31084749dc5b1d), you can tell that a cell is loading by the appearance of a gray stripe to the left of the cell.

Is there any mechanism in Observable Framework by which a cell can know when it's loading, ie. with a CSS class analogous to `observablehq--running` in notebooks, so that it can be styled differently? 

The use case that prompted this is that I have a page with a dataset picker input. Changing their selection causes a new file to be loaded, but while that file is being loaded, the old charts continue to be displayed for a second or two, even though they correspond to the previous selection. It would be nice if they could be visually marked as invalidated or pending so it's clear that the displayed data does not correspond to the new selection.

**Comment by Fil** [comment]
I don't think you can do this at the moment. I'll move this to a feature request.

**Reply by Fil** [reply, marked as answer]
https://github.com/observablehq/framework/issues/1610
